src/coreComponents/constitutive/CMakeLists.txt

```diff
@@ -271,14 +286,11 @@ set( dependencyList ${parallelDeps} events dataRepository functions denseLinearA
 if( ENABLE_PVTPackage )
     set( constitutive_headers
          ${constitutive_headers}
-         fluid/multifluid/compositional/CompositionalMultiphaseFluid.hpp
-         )
+         fluid/multifluid/compositional/CompositionalMultiphaseFluidPVT.hpp )
```

Comment:
Suggest we rename this to `CompositionalMultiphaseFluidPVTPackage` to be obvious what this is.